addon/services/best-language.js

```diff
@@ -1,12 +1,16 @@
 import Ember from 'ember';
 
-const {inject} = Ember;
+const {computed, getOwner} = Ember;
 
 export default Ember.Service.extend({
-  fastboot: inject.service('fastboot'),
+  fastboot: computed(function() {
+    const owner = getOwner(this);
```

Comment:
Is the variable really necessary? We could use `getOwner(this).lookup('service:fastboot');`.

Reply:
True, fixed!